projects/demo/src/modules/components/input-files/examples/1/index.ts

```diff
@@ -24,6 +30,24 @@ export class TuiInputFilesExample1 {
     }
 
     clearRejected(): void {
+        this.removeFile();
         this.rejectedFiles$.next(null);
     }
+
+    makeRequest(file: TuiFileLike): Observable<TuiFileLike | null> {
+        this.loadingFiles$.next(file);
+
+        return timer(1000).pipe(
+            map(() => {
+                if (Math.random() > 0.5) {
```

Comment:
Can it produce flaky screenshot tests?

If yes, don't forget to extend this constant:

https://github.com/Tinkoff/taiga-ui/blob/2029a042a45c04f42af9c581197bb25d392db9fd/projects/demo-integrations/cypress/support/exclusions.ts#L2-L9

Reply:
Thanks! No it can't, cypress test doesn't upload files